pkg/chunk/cassandra/storage_client.go

```diff
@@ -89,7 +89,7 @@ func (cfg *Config) Validate() error {
 	return nil
 }
 
-func (cfg *Config) session(name string) (*gocql.Session, error) {
+func (cfg *Config) session(name, purpose string, registerer prometheus.Registerer) (*gocql.Session, error) {
```

Comment:
Lets not add purpose here, but rather pass in `prometheus.DefaultRegisterer(prometheus.Labels{"schema_start": ...})` to `NewStorageClient`